### PR TITLE
Initial work on codefordc postgres

### PR DESCRIPTION
**Pull request by mkalish**
@NealHumphrey

### CODE REVIEW

**Comment by mkalish** [comment]
This is a working version of creating a shareable postgres container and an docker-compose file for running postgres and pgweb

File: postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Thats it.  You have now created a reusable docker container with your project's data that can
+be shared with your team.
+
+
+## Sample docker-compose.yml
+The docker-compose.yml is an example of how to use the created postgres docker image.  To get started:
+
+1. Copy `docker-compose.yml` into your project.
+2. Replace `codefordc2/postgres` with the name of you image.
```
Comment (trycrmr) [inline postgres/README.md:32]: Did you mean "your" image?

**Review by trycrmr — APPROVED** [review]

**Comment by trycrmr** [comment]
Didn't pull down to validate before merging but makes sense. Pretty cool!

**Comment by NealHumphrey** [comment]
@mkalish Awesome! I will try to use this today, might be able to get a HousingInsights version working in time for c4dc? Depends how long user interviews take today..

**Comment by mkalish** [comment]
@NealHumphrey Awesome, let me know if you run into any issues I can try to help you debug remotely or sometime tonight

**Pull request merged**